src/clients/gists.js

```diff
@@ -103,26 +72,27 @@ function createPopcodeJson(project) {
 }
 
 export async function createFromProject(project, user) {
-  const github = clientForUser(user);
+  const github = gitHub.clientForUser(user);
```

Comment:
Like with the comment about repos/gists, combined `clients`, `repos`, and `services/github` into a single `clients/github` module